Products/CMFPlone/tests/test_okay.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from plone.testing.z2 import Browser
+from Products.CMFPlone.testing import PRODUCTS_CMFPLONE_INTEGRATION_TESTING
+
+import unittest
+
+
+class OkayTest(unittest.TestCase):
+    """Test the OK simple status view.
```

Comment:
this one-line docstring must be written `"""Test the OK simple status view."""` instead and have a clean line above it.